assets/js/script.js

```diff
@@ -11,7 +11,19 @@ let matchHistory = [];
 let lastScoringPlayer = null;
 let lastRotationalServesCounter = 0;
 let lastServe = -1;
-
+function addWinBalls() {
+    const value = parseInt(document.getElementById('winBalls').value);
+    if (!isNaN(value)) {
+        document.getElementById('winBalls').value = value + 1;
+    }
```

Comment:
也许可以在输入框内容无效或为空时设置值为一个默认的整数？

就像这样：


```suggestion
    if (!isNaN(value)) {
        document.getElementById('winBalls').value = value + 1;
    }
    else {
        document.getElementById('winBalls').value = 5;
    }
```

减小函数同理

Reply:
这里可以用条件操作符 `isNaN(value) ? 5 : Number(value) + 1;` 更简单一些

Reply:
> 这里可以用条件操作符 `isNaN(value) ? 5 : Number(value) + 1;` 更简单一些

这里可以直接 `value + 1` 而不用先 `Number(value)` 吧

Reply:
行